### PR TITLE
Update standard to use medic-conf 3.0

### PR DESCRIPTION
**Pull request by kennsippell**
# Description

Medic-conf 3.0 includes support for ES6 + a few breaking changes to the declarative configuration system. The current proposal is in review and detailed at https://github.com/medic/medic-conf/pull/215.

This is a preview of what is required to update standard to medic-conf 3.0. Tests will fail until I actually bump the package.json version, but waiting for a round of code review feedback before I take those steps. Early feedback welcome!

# Code review items

- Readable: Concise, well named, follows the [style guide](https://github.com/medic/medic-docs/blob/master/development/style-guide.md), documented if necessary.
- Documented: Configuration and user documentation on [medic-docs](https://github.com/medic/medic-docs/)
- Tested: Unit and/or e2e where appropriate
- Internationalised: All user facing text
- Backwards compatible: Works with existing data and configuration or includes a migration. Any breaking changes documented in Changes.md.

# License

The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.

### CODE REVIEW

File: config/standard/test/targets.spec.js [hunk 1]
```diff
@@ -3,7 +3,7 @@
 // chai.use(chaiExclude);
 // const assert = chai.assert;
 const assert = require('chai').assert;
-const NootilsManager = require('medic-nootils/src/node/test-wrapper');
+const NootilsManager = require('@kennsippell/medic-nootils/src/node/test-wrapper');
```
Comment (abbyad) [inline config/standard/test/targets.spec.js]: Is this meant to be linking to the medic version instead?

File: config/standard/test/tasks.spec.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
 const _ = require('lodash');
 const assert = require('chai').assert;
-const NootilsManager = require('medic-nootils/src/node/test-wrapper');
+const NootilsManager = require('@kennsippell/medic-nootils/src/node/test-wrapper');
```
Comment (abbyad) [inline config/standard/test/tasks.spec.js]: Link to mainstream version here too?

File: config/standard/test/targets.spec.js (hunk 1 above)
Comment (kennsippell) [reply config/standard/test/targets.spec.js]: Yep - this is normal for testing a package when the change has yet to pass code review. Diana approved the change this morning, so I will now remove it and publish the live `medic-conf 3.0.0` package

File: config/standard/targets.js [hunk 3]
```diff
@@ -364,28 +358,17 @@ module.exports = [
       return immunizationForms.indexOf(r.form) !== -1;
     },
     date: 'reported',
-    emitCustom: function(inst, c, r) {
-      var i, instance;
+    emitCustom: function(emit, inst, c, r) {
```
Comment (kennsippell) [inline config/standard/targets.js]: @eljhkrr Would you help confirm these changes didn't break anything?

File: config/standard/.eslintrc [hunk 4]
```diff
@@ -1,17 +1,17 @@
...
     "ecmaVersion": 6,
     "sourceType": "module"
   },
-  "rules": {
-    "no-undef": "off",
-    "no-unused-vars": "off"
+  "globals": {
+    "contact": true,
+    "reports": true,
+    "lineage": true
```
Comment (eljhkrr) [inline config/standard/.eslintrc]: It would be a good idea to make globals read-only

**Pull request merged**